src/api/lambda.ts

```diff
@@ -67,6 +67,43 @@ export const lambdaCall = (fnName: string, payload: any): Promise<any> =>
     });
   });
 
+
+/**
+ * Adds a message to the SQS alerting queue
+ *
+ * @param fnName - The lambda function name
+ * @param payload - The payload to be sent
+ */
+export const addAlert = async (title: string, message: string, severity: Severity, metadata?: any): Promise<void> => {
+  const preparedMessage = {
+    title,
+    message,
+    severity,
+    metadata,
+    application: process.env.APPLICATION_NAME,
+  };
+
+  const sqs = new AWS.SQS({ apiVersion: '2015-03-31' });
+
+  const params = {
+    MessageBody: JSON.stringify(preparedMessage),
+    QueueUrl: process.env.ALERT_QUEUE_URL as string,
+    MessageAttributes: {
+      None: {
+        DataType: 'String',
+        StringValue: '--',
+      },
+    }
+  };
+
+  sqs.sendMessage(params, (err) => {
+    if (err) {
+      logger.error('[ALERT] Erroed while sending message to the alert sqs queue');
```

Comment:
This means we shouldn't disable the old alerting mechanism yet, right?

Reply:
That is correct, a failure to create the alert should be an alert